data/pigui/modules/settings-window.lua

```diff
@@ -500,7 +492,7 @@ local function showControlsOptions()
 
 	for _,page in ipairs(binding_pages) do
 		ui.text ''
-		ui.withFont(pionillium.medium.name, bindingPageFontSize, function()
+		ui.withFont(pionillium.medium.name, pionillium.medium.size, function()
```

Comment:
If possible, I'd appreciate it if you condensed these withFont lines to `ui.withFont(pionillium.heading, function()` - there's no need to manually fetch the size and name separately and I'd prefer to use the semantic breakpoints rather than the medium/medlarge/large "size" breakpoints going forwards.

Reply:
I don't understand the system - on what basis to assign a style, so if you don't mind, I'll leave it as it is, just don't pass name and size separately.

Reply:
That's fine by me - at the moment it's just a rough match between existing size and the semantic purpose of the text.